src/ui/public/vis/response_handlers/legacy_table.js

```diff
@@ -0,0 +1,79 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import get from 'lodash.get';
+import moment from 'moment';
+import AggConfigResult from '../../vis/agg_config_result';
+
+function keyRowsByValue(rows, key) {
+  return rows.reduce((acc, row) => {
+    const { [key]: splitValue, ...rest } = row;
+    acc[splitValue] = [...(acc[splitValue] || []), rest];
+    return acc;
+  }, {});
+}
+
+function formatSplitFieldValue(aggConfig, value) {
+  if (get(aggConfig, 'params.field.type') === 'date') {
+    return moment(parseInt(value)).format('YYYY-MM-DD');
+  }
+  return value;
+}
```

Comment:
Someone *must* have written a util of some type to handle this elsewhere. I grepped around and was unable to find how we were previously formatting these dates in table titles, so added this naïve implementation for now. But I'd greatly appreciate if someone could point me in the right direction so that I can remove this!